lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +22,96 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
```

Comment:
Does it still need `container-vertical`?

Reply:
> Does it still need `container-vertical`?

Nope: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/794bc0895d8004448a8bc76d097cc22f963463f1

Reply:
> Nope: 794bc08

That removed the wrong one. Though I guess that one wasn't needed anymore either.

Reply:
> > Nope: [794bc08](https://github.com/sonatype/sonatype-react-shared-components/commit/794bc0895d8004448a8bc76d097cc22f963463f1)
> 
> That removed the wrong one. Though I guess that one wasn't needed anymore either.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/79bc7e182bee3c81f39445a3a330b363f8f8e065